frontend/src/job/SpecificJobsList.vue

```diff
@@ -32,89 +32,65 @@
                 </b-table-column>
             </template>
             <template slot="detail" slot-scope="props" v-if="props.row.children.length > 0">
-                <SpecificJobsList :jobIDs="getChildrenIDs(props.row.children)" />
+                <SpecificJobsList v-if=childJobsLoaded :jobs="childJobs" />
+                <div v-else>
+                    Loading child jobs...
+                </div>
             </template>
         </b-table>
-        </div>
     </section>
 </template>
 
 <script lang="ts">
 import { Component, Vue, Prop } from 'vue-property-decorator';
-import moment from 'moment';
-import { getJobDetails, getJobList } from './JobClient';
-import { Job } from './Job';
-import { ChildJob } from './ChildJob';
+import { getJobDetails, getJobList, Job } from './JobClient';
 import { showError } from '@/util/Notifier.ts';
 
 @Component
 export default class SpecificJobsList extends Vue {
-    @Prop(Array) jobIDs!: string[];
     @Prop(Array) jobs!: Job[];
-    jobList: Job[] = [];
+    jobList: Job[] = this.jobs;
+    childJobs: Job[] = [];
```

Comment:
du brauchst ein ChildJobs für jeden Job. Sonst werden bei mehreren öffnen vvon details falsche Sachen angezeigt.

Reply:
> detail anzeige noch verbugged

Ich brauche mehr Infos hier...^^ Was genau ist verbuggt?

>du brauchst ein ChildJobs für jeden Job. Sonst werden bei mehreren öffnen vvon details falsche Sachen angezeigt.

Auch hier weiss ich nicht genau wie das zu testen ist.

Gib mal für beides Reproduktionsschritte bitte.

Reply:
Mit mehreren Jobs in der Liste:
Details zu einem Job öffnen, dann vom zweitem (IDs merken der Kinder)
Dann erstes schließen und wieder Öffnen (ID ist dann vom Kind vom anderen)

Reply:
un verbuggt ist genau das, war als Titel meines review gedacht nicht noch zusätzlicher Bug